src/Tools/dotnet-counters/Program.cs

```diff
@@ -38,6 +48,18 @@ internal class Program
                 "The number of seconds to delay between updating the displayed counters.",
                 new Argument<int>(defaultValue: 1) { Name = "refresh-interval" });
 
+        private static Option ExportFormatOption() => 
+            new Option(
+                new[] { "--format" },
+                "The format of exported counter data.",
+                new Argument<string>(defaultValue: "csv") { Name = "format" });
```

Comment:
Echoing my other comment, if you make this `Argument<CountersExportFormat>()` where `CountersExportFormat` is an enum, S.CommandLine will automatically convert strings into the enum values and error on invalid input.